backend/internal/service/handler/review.go

```diff
@@ -0,0 +1,29 @@
+package handler
+
+import (
+	"platnm/internal/storage"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type ReviewHandler struct {
+	reviewRepository storage.ReviewRepository
+}
+
+func NewReviewHandler(reviewRepository storage.ReviewRepository) *ReviewHandler {
+	return &ReviewHandler{
+		reviewRepository,
+	}
+}
+
+func (h *ReviewHandler) GetReviewsByUserID(c *fiber.Ctx) error {
+	id := c.Params("id")
+	review, err := h.reviewRepository.GetReviewsByUserID(c.Context(), id)
+
+	if err != nil {
+		print(err.Error(), "from transactions err ")
```

Comment:
you dont have to log error here, we added central error handling which will log any error that is returned to the handler

[http_error.go](https://github.com/GenerateNU/platnm/blob/main/backend/internal/errs/http_error.go#L64)

Reply:
gone now I believe